clayui.com/content/docs/examples.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Examples'
+order: 5
+---
+
+import ClayLayout from '@clayui/layout';
+
+## What is Storybook and how do we use it?
+
+Storybook is a user interface development environment and playground for UI components.
+We use it to create components independently and showcase components interactively in an isolated development environment.
+A Storybook is a collection of stories. Each story represents a single visual state of a component.
+
+## How you can leverage Storybook to make using Clay easier
+
+Whenever you have doubts about how a component is implemented or used, checking out its story in Storybook
+is a great way of getting valuable information about how we envisioned that component be used.
+
+You can use Storybook for inspiration, as well as guidelines, for your use case as well as getting insight into the composition of a component.
+
+## Examples/Demos
+
+Here are our demos we have in Storybook that utilize Clay components showcasing some common use cases.
```

Comment:
Instead of using screenshots, we could try adding in iframe examples from the storybook. Or we could also just directly link to the examples.

Full page url we could iframe, https://storybook.clayui.com/iframe.html?id=demos-templates--list-page

Reply:
I'll propose a new solution using iframes. Something to bring value until we get some illustrations going, I guess.

Reply:
Don't overdo it for now.. simple links would suffice to get started. The screenshot/illustration idea is just a way to sweeten the deal and make links more appealing.